src/themes/default/components/core/blocks/Footer/Footer.vue

```diff
@@ -149,69 +159,81 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import '~theme/css/global_vars';
-$gray-secondary: map-get($colors, gray-secondary);
+  @import '~theme/css/global_vars';
+  $gray-secondary: map-get($colors, gray-secondary);
 
   .newsletter-content {
     align-items: center;
   }
+
   .icon {
     transition: 0.3s all;
   }
+
   .social-icon {
     display: inline-block;
     width: 40px;
     height: 40px;
     border-radius: 50%;
-    &:hover .icon,
-    &:focus .icon,
-    &:active .icon {
-      fill: $gray-secondary;
+    &:hover,
+    &:focus,
+    &:active {
+      .icon {
+        fill: $gray-secondary;
+      }
     }
   }
 
   .links {
+    padding-left: 0;
```

Comment:
use atomic class instead please (we use them to have minimal css footprint without repeating things like 'padding-left: 0')

Reply:
Sure, fixed.